js/components/Instructions.vue

```diff
@@ -1,91 +1,108 @@
 <template>
   <ul id="vizr-instructions" class="list-unstyled" v-if="showInstructions">
     <li v-if="showNoChartInstructions" data-no-charts>{{ messages.mainDescription }}</li>
-    <li v-if="showNoChartInstructions" data-no-charts>{{ messages.threeComponents }}
+    <li v-if="showNoChartInstructions" data-no-charts>
+      {{ messages.threeComponents }}
       <ul>
         <li>{{ messages.chartComponents.recordFilter }}</li>
         <li>{{ messages.chartComponents.timePoint }}</li>
         <li>{{ messages.chartComponents.targetProjection }}</li>
       </ul>
     </li>
     <li>
-      <a href="#"
-         data-toggle="collapse"
-         data-target="#collapseMore"
-         aria-expanded="false"
-         aria-controls="collapseMore"
-      >{{ messages.moreInfo.title }}</a>
+      <a
+        href="#"
+        data-toggle="collapse"
+        data-target="#collapseMore"
+        aria-expanded="false"
+        aria-controls="collapseMore"
+        >{{ messages.moreInfo.title }}</a
```

Comment:
@lawhead already mentioned dangling `>` but that closing tag looks extra weird.

Reply:
I found a few cases like this that I fixed by manually wrapping the whole closing tag onto the next line. It left them alone afterwards.